.travis.yml

```diff
@@ -43,8 +42,10 @@ install:
   # nose is already installed
   # numpy is already installed
   - pip install -r travis-ci-requirements
-  - pip install coveralls
+  - if [[ $TRAVIS_PYTHON_VERSION == "2.7_with_system_site_packages" ]]; then pip install kiwisolver; fi
```

Comment:
you can use the pip enviroment markers (https://pip.pypa.io/en/latest/reference/pip_install.html#requirement-specifiers)